assets/css/index.scss

```diff
@@ -1,11 +1,11 @@
-// Import the US Web Design System minified CSS. This is required by asset pipeline.
+// Import the US Web Design System SASS.
 
+@import "uswds";
```

Comment:
Hello SCSS! Now we can theme our USWDS SCSS by overriding variables and add our own!